providers/openfeature-provider-flagd/tests/test_errors.py

```diff
@@ -27,10 +29,11 @@ def create_client(provider: FlagdProvider):
     ],
 )
 def test_file_load_errors(file_name: str):
+    path = os.path.abspath(os.path.join(os.path.dirname(__file__), "./flags/"))
```

Comment:
For me it was easier to just load one provider as a project, rather than the whole sdk, but the tests where broken, due to paths issues, with this, both versions of running the tests work